sql/dql/basic-queries/select.md

```diff
@@ -31,23 +31,45 @@ aspects:
 ---
 ## Content
 
-To get all of the data from a table, you can use a `SELECT` statement.
+Consider the following table named `pokemon`:
```

Comment:
maybe worth adding a quick intro:

"The `SELECT` command is used to retrieve data from tables.."

Reply:
Added quick intro